prototype2/common/TimeString.cpp

```diff
@@ -0,0 +1,16 @@
+/** Copyright (C) 2017 European Spallation Source ERIC */
+
+#include <common/TimeString.h>
+
+std::string timeString() {
+  char cStartTime[50];
+  time_t rawtime;
+  struct tm *timeinfo;
+  time(&rawtime);
+  timeinfo = localtime(&rawtime);
+  strftime(cStartTime, 50, "%Y-%m-%d-%H-%M-%S", timeinfo);
```

Comment:
Used to be %Y%m%d-%H%M%S which I think make a better distinction between 
date and time and is still sortable.

Reply:
OK, I'll change it.